microk8s-resources/actions/dashboard.yaml

```diff
@@ -167,18 +167,14 @@ spec:
     spec:
       containers:
         - name: kubernetes-dashboard
-          image: kubernetesui/dashboard:v2.0.0-beta5
+          image: kubernetesui/dashboard:v2.0.0-rc5
```

Comment:
We may want to have this as a separete PR so it does not get blocked by the rest of the work we are doing here.